src/bloaty.h

```diff
@@ -411,6 +433,9 @@ struct RollupOutput {
   // When we are in diff mode, rollup sizes are relative to the baseline.
   bool diff_mode_ = false;
 
+  // Print Total line 
+  mutable bool capacity_total_ = true;
```

Comment:
The `mutable` keyword should be avoided as it disables essential compiler safety checks. I recommend restructuring the code so that it isn't required (see my other comment).

I would suggest using a more descriptive name for this flag, such as `should_print_total_`.